provisioning/service/src/Authentication/ServiceAuthenticationWithSharedAccessPolicyKey.cs

```diff
@@ -38,11 +34,6 @@ public string Key
 
         public ServiceConnectionStringBuilder Populate(ServiceConnectionStringBuilder provisioningConnectionStringBuilder)
         {
```

Comment:
Debug.Assert(provisioningConnectionStringBuilder != null, $"nameof(provisioningConnectionStringBuilder)} cannot be null. Validate parameters upstream.");